packages/terra-clinical-result/src/proptypes/statusPropTypes.js

```diff
@@ -0,0 +1,18 @@
+import PropTypes from 'prop-types';
+
+/**
+ * Enum for possible Result Statuses.
+ * values mapped from http //hl7.org/fhir/2018Sep/valueset-observation-status.html
```

Comment:
I assume this is why they aren't capitalized like type/interpretation. Do we need to use these FHIR concepts directly?

Reply:
These are statuses integrated directly from HL7, vs the other enums that are all caps are from current Millennium conventions.